assets/js/langpack/ru.json

```diff
@@ -0,0 +1,151 @@
+{
+  "token": {
+
+    "The makers' choice for sysadmins, developers and desktop users.":
+    "The makers' choice for sysadmins, developers and desktop users.",
```

Comment:
This string is not translated together with many ones at the bottom. Can you check again? Thank you!